changelogs/fragments/bgp_af_fam_284.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - "iosxr_bgp_neighbor_address_family - add extra supported values to afi attribute."
```

Comment:
Can this be a bit more descriptive? Maybe add the new values?

Reply:
sure